internal/display/logs.go

```diff
@@ -144,7 +144,7 @@ func (v *logView) typeDesc() (typ, desc string) {
 	return typ, desc
 }
 
-func (r *Renderer) LogList(logs []*management.Log, ch <-chan []*management.Log, silent bool) {
+func (r *Renderer) LogList(logs []*management.Log, silent bool) {
```

Comment:
The behavior you're describing is not caused by this change, it existed prior. Though I do agree with the general sentiment.

Implementing isn't exactly trivial because the render functions are abstracted away from knowing the filters and other arguments. The best we could do here is to provide a generalized message that would apply to alls situations regardless of provided criteria.

Reply:
> The behavior you're describing is not caused by this change, it existed prior. Though I do agree with the general sentiment.

Well we can still leave the codebase a bit better than it was, and also improve DX considering we're now touching this part.

To implement this we can check inside `listLogsCmd()` if we have any filters selected and pass it to the `LogList() `func.

```go
hasFilter := inputs.Filter != ""
cli.renderer.LogList(list, !cli.debug, hasFilter)
```

then inside the `LogList()` we can tweak the if `len(logs) == 0` check as follows:

```go
if len(logs) == 0 {
		if hasFilter {
			r.Output("No logs available matching filter criteria.\n\n")
		} else {
			r.EmptyState(resource)
		}
		r.Infof("To generate logs, run a test command like 'auth0 test login' or 'auth0 test token'")
		return
	}
```

This will have the desired behavior as described above. 

```
go run cmd/auth0/main.go logs ls -f "type:f" 

=== example.auth0.com logs

No logs available matching filter criteria.

 ▸    To generate logs, run a test command like 'auth0 test login' or 'auth0 test token'
```

Reply:
Let's address in a follow-up PR that's more focused.